srcs/server/src/app.module.ts

```diff
@@ -1,21 +1,34 @@
-import { Module } from '@nestjs/common';
+import { CacheModule, Module } from '@nestjs/common';
 import { AuthModule } from './auth/auth.module';
 import { UserModule } from './user/user.module';
 import { PrismaModule } from './prisma/prisma.module';
 import { ConfigModule } from '@nestjs/config';
 import { FriendshipModule } from './friendship/friendship.module';
 import { MatchModule } from './match/match.module';
 import { RatingModule } from './rating/rating.module';
+import { GameModule } from './game/game.module';
+import * as redisStore from 'cache-manager-redis-store';
+import { RedisModule } from './redis/redis.module';
 
 @Module({
   imports: [
-    ConfigModule.forRoot(),
+    ConfigModule.forRoot({
+      envFilePath: `.env.${process.env.NODE_ENV}`,
+    }),
+    CacheModule.register({
```

Comment:
do we know which database does this use ? Meaning is it in conflict with our databases ?

Reply:
This was actually a test. It is cleaner when it comes to testing as the intention is to give specific env variables to the back depending on the environment but as I stopped websocket testing for the moment it is not very usefull. We will have to modify it when we will test sockets.